californium-core/pom.xml

```diff
@@ -37,6 +37,35 @@
 	<build>
 		<plugins>
 			<plugin>
+				<groupId>org.apache.felix</groupId>
+				<artifactId>maven-bundle-plugin</artifactId>
+				<extensions>true</extensions>
+				<configuration>
+					<instructions>
+						<Export-Package>
+							org.eclipse.californium.core,
```

Comment:
They are available from maven central but again, IMHO we should not declare those as dependencies because they would create a transitive dependency on OSGi artifacts. So, IMHO we (i.e. you :-)) need to refrain to setting the minimum version information explicitly in the maven bundle plugin config ...